src/content/reference/okteto-cli.mdx

```diff
@@ -268,17 +269,27 @@ If you need to destroy external resources (like s3 buckets or other Cloud resour
 $ okteto destroy
 ```
 
+The `okteto destroy` command will only destroy resources matching the specified name. In a monorepo scenario, where you might have several manifests, you need to specify the correct manifest to destroy the specific development environment. This ensures that the appropriate destroy commands, as defined in your Okteto manifest, are executed within a particular scope.
```

Comment:
"The specified environment name"? (Not sure how we call it but I feel that just using "name" might be too vague)

Reply:
Good call out. I think "environment name" is a good clarification.